src/town_cmd.cpp

```diff
@@ -2272,6 +2272,14 @@ bool GenerateTowns(TownLayout layout)
 	return false;  // we are still without a town? we failed, simply
 }
 
+void ExpandTowns()
```

Comment:
this function is small enough that it might be worth considering not having it at all, just sticking it in the loop in town_gui.cpp ?

but if not, requires some doxygen comments :)

Reply:
I agree, the whole loop could be moved to the OnClick handler.

Reply:
Fixed this too, thanks for the suggestions!